src/Altinn.Profile/Altinn.Profile.csproj

```diff
@@ -13,6 +13,10 @@
     <PackageReference Include="Azure.Extensions.AspNetCore.Configuration.Secrets" Version="1.4.0" />
     <PackageReference Include="JWTCookieAuthentication" Version="4.0.4" />
     <PackageReference Include="Altinn.Common.PEP" Version="4.1.2" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="9.0.2">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
```

Comment:
We can have this reference in the integration project instead. That is where we have dependencies to EF. Just remember to remove the `<PrivateAssets>` tag. That makes the assets available for the Profile (web) project as well for when we need to run the `dotnet ef` commands.

Reply:
Then we need a startup-file in the integration-project as well, I think

Reply:
We would still need to run the command with the startup project as input. It will have access to all necessary assets through the dependency to the integration porject. Just need to remove the PrivateAssets tag.

Reply:
It worked without adding a program.cs in the integrations-project